src/data/inchi-dictionary.ts

```diff
@@ -2,430 +2,825 @@ import { INCHIProps } from '../interfaces/utils';
     molecularWeight: 79.06,
   },
+
   AmmoniumBromide: {
+    CAS: '12124-97-9',
     inchiString: 'InChI=1S/BrH.H3N/h1H;1H3',
     inchiKey: 'SWLVFNYSXGMGBS-UHFFFAOYSA-N',
     molecularWeight: 97.94,
   },
+
   AmmoniumCarbonate: {
+    CAS: '',
     inchiString: 'InChI=1S/CH2O3.2H3N.H2O/c2-1(3)4;;;/h(H2,2,3,4);2*1H3;1H2',
     inchiKey: 'MMVILKIFGQPOLE-UHFFFAOYSA-N',
     molecularWeight: 114.1,
   },
+
   AmmoniumCarbonateCarbamate: {
+    CAS: '8000-73-5',
     inchiString:
       'InChI=1S/CH3NO2.CH2O3.2H3N/c2*2-1(3)4;;/h2H2,(H,3,4);(H2,2,3,4);2*1H3',
     inchiKey: 'YNSUIDWLZCFLML-UHFFFAOYSA-N',
     molecularWeight: 157.13,
   },
+
   AmmoniumCarbonateSesqui: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 272.21,
   },
+
   AmmoniumChlorideSalammoniac: {
+    CAS: '12125-02-9',
     inchiString: 'InChI=1S/ClH.H3N/h1H;1H3',
     inchiKey: 'NLXLAEXVIDQMFP-UHFFFAOYSA-N',
     molecularWeight: 53.49,
   },
+
   AmmoniumChloroplatinate: {
+    CAS: '16919-58-7',
     inchiString: 'InChI=1S/6ClH.2H3N.Pt/h6*1H;2*1H3;/q;;;;;;;;+4/p-4',
     inchiKey: 'PCCGQTHFYHJATL-UHFFFAOYSA-J',
     molecularWeight: 443.87,
   },
+
   AmmoniumChloroplatinite: {
+    CAS: '13820-41-2',
     inchiString: 'InChI=1S/4ClH.2H3N.Pt/h4*1H;2*1H3;/q;;;;;;+2/p-2',
     inchiKey: 'QJIMNDWDOXTTBR-UHFFFAOYSA-L',
-    molecularWeight: 372.97, // 373.0 # https://pubchem.ncbi.nlm.nih.gov/compound/16211508
+    molecularWeight: 372.97,
   },
+
   AmmoniumChlorostannate: {
+    CAS: '',
     inchiString: 'InChI=1S/6ClH.2H3N.Sn/h6*1H;2*1H3;/q;;;;;;;;+4/p-4',
     inchiKey: 'UDMAVAHFQCVSSE-UHFFFAOYSA-J',
     molecularWeight: 367.5,
   },
+
   AmmoniumChromate: {
+    CAS: '',
     inchiString: 'InChI=1S/Cr.2H3N.4O/h;2*1H3;;;;/q;;;;;2*-1/p+2',
     inchiKey: 'MFFLHUNPSHBKRG-UHFFFAOYSA-P',
     molecularWeight: 152.07,
   },
+
   AmmoniumCyanide: {
+    CAS: '12211-52-8',
     inchiString: 'InChI=1S/CN.H3N/c1-2;/h;1H3/q-1;/p+1',
     inchiKey: 'ICAIHGOJRDCMHE-UHFFFAOYSA-O',
     molecularWeight: 44.06,
   },
+
   AmmoniumDichromate: {
+    CAS: '7789-09-5',
     inchiString: 'InChI=1S/2Cr.2H3N.7O/h;;2*1H3;;;;;;;/q;;;;;;;;;2*-1/p+2',
     inchiKey: 'JOSWYUNQBRPBDN-UHFFFAOYSA-P',
     molecularWeight: 252.06,
   },
+
   AmmoniumFerrocyanide: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 392.19,
   },
+
   AmmoniumFluoride: {
+    CAS: '12125-01-8',
     inchiString: 'InChI=1S/FH.H3N/h1H;1H3',
     inchiKey: 'LDDQLRUQCUTJBB-UHFFFAOYSA-N',
     molecularWeight: 37.04,
   },
+
   AmmoniumFluorideAcid: {
+    CAS: '1341-49-7',
     inchiString: 'InChI=1S/2FH.H3N/h2*1H;1H3',
     inchiKey: 'KVBCYCWRDBDGBG-UHFFFAOYSA-N',
     molecularWeight: 57.04,
   },
+
   AmmoniumFormate: {
+    CAS: '540-69-2',
     inchiString: 'InChI=1S/CH2O2.H3N/c2-1-3;/h1H,(H,2,3);1H3',
     inchiKey: 'VZTDIZULWFCMLS-UHFFFAOYSA-N',
     molecularWeight: 63.06,
   },
+
   AmmoniumHydrosulfide: {
+    CAS: '',
```

Comment:
12124-99-1 is given as CAS in PubChem but does not correspond to any compound in "Sigmaaldrich"

Reply:
Molecular weight does not match so let's leave it.

Reply:
`12124-99-1` is preferred. Ref.: https://pubchem.ncbi.nlm.nih.gov/compound/25515#section=Molecular-Formula

Reply:
@sumitnegi7 did you apply this change ?

Reply:
Didn't work on the inchi PR yesterday @qur786  bringing the change now.